CHANGELOG/CHANGELOG-3.1.md

```diff
@@ -0,0 +1,99 @@
+# Release notes for v3.1.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v3.0.0
+
+## Changes by Kind
+
+### Feature
+ - Introduces `HonorPvReclaim` feature gate that ensures PV honors the Reclaim policy irrespective of deletion order. ([#679](https://github.com/kubernetes-csi/external-provisioner/pull/679), [@deepakkinni](https://github.com/deepakkinni))
```

Comment:
It is "HonorPVReclaimPolicy".

Reply:
it is actually wrongly mentioned in the PR release note itself :(..

Reply:
I've fixed it in this PR.